### PR TITLE
add parsing method

### PR DESCRIPTION
**Pull request by kate-shine**
Parsing method presented in a readme didn't exist :)

### CODE REVIEW

**Comment by ocrickard** [comment]
Wow, I'm terrible, so sorry for missing this! Will review.

File: src/model/ResourceList.rs [hunk 1]
```diff
@@ -316,8 +316,12 @@ impl ResourceList<'_> {
     }
 
     pub fn resource(&self) -> Option<ResourceListEnum> {
-        let fhir_type = self.value["resourceType"].as_str().unwrap();
-        match fhir_type {
+        let fhir_type = self.value.get("resourceType")?;
+        if !fhir_type.is_string() {
```
Comment (ocrickard) [inline src/model/ResourceList.rs:320]: Is this required to pass a test?

**Comment by ocrickard** [comment]
This project is changing ownership to some more active maintainers, please submit this pull request to the new repo here: https://github.com/fhirproject/fhir-rs

Thank you, and good luck!!

**Comment by kate-shine** [comment]
Thank you, but we ended up developing our own macro generated solution at the end, which will get published soon :)

**Pull request closed without merging**